packages/schema-org/src/composables/injectSchemaOrg.ts

```diff
@@ -0,0 +1,16 @@
+import { inject } from 'vue'
+import { PROVIDE_KEY } from './createSchemaOrg'
+import type { SchemaOrgVuePlugin } from './createSchemaOrg'
+
+export function injectSchemaOrg() {
```

Comment:
hopefully with this and the createSchemaOrg being shared the injection will work